src/inspector/inspectorhandler.cpp

```diff
@@ -544,6 +544,25 @@ static QList<InspectorCommand> s_commands{
                        return obj;
                      }},
 
+    InspectorCommand{"is_feature_enabled", "Check if a feature is enabled", 1,
```

Comment:
What is the difference between enabling a feature and flipping a feature on? A comment might be helpful here (or maybe there is some documentation that I missed).

Reply:
Learned this the hard way. We use the term "flipping" and "flipped" to describe features that were toggled on/off manually (either through the UI, or the inspector), otherwise, the list of flipped on/off features is empty. 

The command I have added is simply to see if the feature is on or off,.

Will add a comment as well!